packages/protocol/contracts/stability/Reserve.sol

```diff
@@ -394,6 +443,77 @@ contract Reserve is
     return _transferGold(to, value);
   }
 
+  /**
+   * @notice Transfer ERC20 token to a whitelisted address subject to reserve spending limits
+   * if the limit is set, othersise the limit is 100%.
+   * @param erc20TokenAddress The token address you're transferring.
+   * @param to The address that will receive the ERC20 token.
+   * @param value The amount of ERC20 token to transfer.
+   * @return Returns true if the transaction succeeds.
+   */
+  function transferErc20Token(address erc20TokenAddress, address payable to, uint256 value)
+    external
+    returns (bool)
+  {
+    require(
+      erc20TokenAddress != registry.getAddressForOrDie(GOLD_TOKEN_REGISTRY_ID),
+      "token address should not equal to celo"
+    );
+    require(isSpender[msg.sender], "sender not allowed to transfer Reserve funds");
+    require(isOtherReserveAddress[to], "can only transfer to other reserve address");
+    uint256 spendingLimitForThisToken;
+    if (FixidityLib.unwrap(erc20DailySpendingRatio[erc20TokenAddress]) > 0) {
+      uint256 currentDay = now / 1 days;
+      if (currentDay > lastSpendingDays[erc20TokenAddress]) {
+        uint256 balance = getErc20TokenBalance(erc20TokenAddress);
+        lastSpendingDays[erc20TokenAddress] = currentDay;
+        spendingLimitForThisToken = erc20DailySpendingRatio[erc20TokenAddress]
+          .multiply(FixidityLib.newFixed(balance))
+          .fromFixed();
+      }
+      require(spendingLimitForThisToken >= value, "Exceeding spending limit");
+      spendingLimitForThisToken = spendingLimitForThisToken.sub(value);
+      return _transferErc20Token(erc20TokenAddress, to, value);
+    } else {
+      return _transferErc20Token(erc20TokenAddress, to, value);
+    }
+  }
+
+  /**
+   * @notice Transfer unfrozen ERC20 token to any address.
```

Comment:
From my understanding, an ERC20/collateral asset cannot be frozen. So this description should not say unfrozen